config/vendor-api-v1.yml

```
@@ -380,7 +380,7 @@ components:
       properties:
         id:
           type: string
-          description: The unique ID of this application
+          description: The unique ID of this application. This identifies an application to a single course, also known as an application choice. Candidates can have up to three application choices for one application form.
```

We don't really have a concept of an application form in the API, not sure if introducing this here will cause even more confusion

---

maybe we just go with `form`

---

Wouldn't application, rather than form, be easier to understand?

---

I think keeping `form` out could be good?

```suggestion
          description: The unique ID of this application. This identifies an application to a single course, also known as an application choice. Candidates can submit up to three application choices at a time.
```